docker/Dockerfile

```diff
@@ -1,17 +1,21 @@
 
 FROM ubuntu:23.10
 ARG VERSION="dev"
-
-RUN apt-get update && \
- apt-get install --no-install-recommends -q --assume-yes openjdk-21-jre-headless=21* libjemalloc-dev=5.* adduser=3* && \
- apt-get clean  && \
- rm -rf /var/lib/apt/lists/*  && \
- # Ubuntu 23.10 comes with an "ubuntu" user with uid 1000. We need 1000 for besu.
- userdel ubuntu 2>/dev/null || true && rm -rf /home/ubuntu && \
- # Ensure we use a stable UID for besu, as file permissions are tied to UIDs.
- adduser --uid 1000 --disabled-password --gecos "" --home /opt/besu besu && \
-    chown besu:besu /opt/besu && \
-    chmod 0755 /opt/besu
+ENV NO_PROXY_CACHE="-o Acquire::BrokenProxy=true -o Acquire::http::No-Cache=true -o Acquire::http::Pipeline-Depth=0"
+
+# Clean apt cache
+RUN apt-get clean  && \
+  rm -rf /var/cache/apt/archives/* /var/cache/apt/archives/partial/*  && \
+  rm -rf /var/lib/apt/lists/*  && \
```

Comment:
since we should assume that the base ubuntu image is good, and the following commands have the cache disable, the cleanup could be moved at the end of the command list, so the image will be smaller.

Reply:
You are right I've changed it to do it after the installation